app/controllers/assets_controller.rb

```diff
@@ -116,6 +116,9 @@ def mass_edit
     end
     @assets = [@user.assets.where(id: params[:assets])].flatten if params[:assets] # expects comma seperated list of ids
     @assets = @user.assets unless @assets.present?
+
+    AssetNotificationJob.set(wait: 10.minutes).perform_later(asset_ids: @assets.map(&:id), user_id: @user.id)
```

Comment:
Kind of confused by this — `@user` is the uploading user — but we want the notification to go to all the users in `user.followers` — the idea is that when I upload, the 25 people following me will each get an email. So this is where `Asset.notify_followers` should be called.